File: ReactNative/src/modules/todo/services/get-todo.service.ts
import { ITodo } from '../../../@types/todo';
import { TokenAxios } from '../../common/utils/token-axios';
import { TODO_URL } from '../../common/utils/url';

export async function requestGetTodo(idx: number): Promise<ITodo> {
  const AXIOS = await TokenAxios();
  return await AXIOS.get(`${TODO_URL}/${idx}`);
}

File: flutter_todo/lib/src/modules/todo/services/get_todo_service.dart
import 'package:flutter_todo/src/modules/common/utils/token_dio.dart';
import 'package:flutter_todo/src/modules/common/utils/url.dart';
import 'package:flutter_todo/src/modules/todo/models/todo_model.dart';

Future<Todo> requestGetTodo(int idx) async => (await tokenDio())
    .get('$todoUrl/$idx')
    .then((response) => Todo.fromJson(response.data['todo']))
    .catchError((error) => throw error);
